src/components/swapv2/LimitOrder/LimitOrderForm.tsx

```diff
@@ -304,28 +306,32 @@ const LimitOrderForm = function LimitOrderForm({
   )
 
   const inputError = useMemo(() => {
-    if (!inputAmount) return
```

Comment:
only add try catch

Reply:
<img width="452" alt="image" src="https://user-images.githubusercontent.com/23650671/212808236-22000216-f37f-4d30-971c-7e6120ff8a44.png">
dont need to add comment bro =))